src/app/public/styles/themes/modern/_buttons.scss

```diff
@@ -77,6 +77,16 @@
     );
   }
 
+  .sky-btn-icon-borderless {
+    @include sky-theme-modern-button-variant(
+      $sky-theme-modern-font-deemphasized-color,
```

Comment:
Is this color correct? It's barely perceptible in the visual tests.

Reply:
the default theme version is super light, that's why we darkened it for modern theme. it should be the normal deemphasized color, which looks like it is to me

Reply:
tho maybe i am looking at the wrong button?